fresh/centrifugo-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "bundles": {
+        "Fresh\\CentrifugoBundle\\FreshCentrifugoBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```

Comment:
Is this really needed? 

Does your bundle require this exact file path?

Reply:
No, it is not required. It is just a small help. It creates a directory `src/Service/Centrifugo/ChannelAuthenticator/` where I recommend to put channel authenticators. I use Centrifugo for a quite long time on several projects. And everywhere there are private channels. And to get access to private channel user should be granted with additional token, so ChannelAuthenticators are quite popular things. I decided to put it to recipe, so new developers would notice this directory in their projects, then take a look in the documentations about private channels https://github.com/fre5h/CentrifugoBundle/blob/master/Resources/docs/authentication.md#create-own-channel-authenticator and so would have an easier setup of my bundle.

Reply:
Im not convinced this is more helpful than annoying to be honest. 

A user may have decided to put all their authentication related stuff in `src/Auth`. Now this recipe is trying to force a coding standard on that user that isnt really needed.